src/Adapter/MSTest.TestAdapter/Execution/ExceptionHelper.cs

```diff
@@ -77,18 +77,22 @@ internal static class ExceptionHelper
         bool first = true;
         while (stackTraces.Count != 0)
         {
-            result.AppendFormat(
-                    CultureInfo.CurrentCulture,
-                    "{0} {1}{2}",
-                    first ? string.Empty : (Resource.UTA_EndOfInnerExceptionTrace + Environment.NewLine),
-                    stackTraces.Pop(),
-                    Environment.NewLine);
+            if (!first)
+            {
+                result.Append(utaEndOfInnerExceptionTrace);
```

Comment:
Maybe `result.AppendLine(Resource.UTA_EndOfInnerExceptionTrace)` so that we still respect CurrentUICulture for resource restrieval?